src/features/orbitalfeature.jl

```diff
@@ -0,0 +1,77 @@
+include("abstractfeatures.jl")
+using DataFrames
+using ..ChemistryFeaturization: elements
+using ..ChemistryFeaturization.AbstractType: AbstractCodec, AbstractAtoms
+using ..ChemistryFeaturization.Codec: SimpleCodec
+using ..ChemistryFeaturization.Utils.OrbitalFeatureUtils:
+    valence_shell_config, _indexorbital
+using SparseArrays
+
+struct OrbitalFeatureDescriptor <: AbstractEnvironmentFeatureDescriptor
```

Comment:
Would this be better handled at the `featurization` level instead? Also, I don't think you can explicitly mention how long a row/column of a sparse matrix (or even a sparse vector for that matter) can be (which is what the default encoding scheme finally produces here).

Reply:
Ahh right okay I forgot that was how sparse matrices work. Yeah that should be fine then. We'll probably have to enforce it by desparsifying at the featurization level since there probably should be a well-defined output shape and certainly if you ever concatenate something onto the end of this, you have to make that dimension explicit.